$MirOS: ports/databases/postgresql/patches/patch-src_interfaces_ecpg_pgtypeslib_datetime_c,v 1.1.7.1 2005/03/18 15:44:16 tg Exp $
--- src/interfaces/ecpg/pgtypeslib/datetime.c.orig	Wed Oct  4 02:29:48 2006
+++ src/interfaces/ecpg/pgtypeslib/datetime.c	Thu Feb 22 20:25:47 2007
@@ -118,8 +118,8 @@ PGTYPESdate_to_asc(date dDate)
 void
 PGTYPESdate_julmdy(date jd, int *mdy)
 {
-	int			y,
-				m,
+	time_t			y;
+	int			m,
 				d;
 
 	j2date((int) (jd + date2j(2000, 1, 1)), &y, &m, &d);
